focus_android/focus_android.model.lkml

```diff
@@ -1,7 +1,7 @@
 connection: "telemetry"
 label: "Firefox Focus for Android"
 include: "//looker-hub/focus_android/explores/*"
-include: "//looker-hub/focus_android/dashboards/*"
-include: "views/*"
-include: "explores/*"
-include: "dashboards/*"
\ No newline at end of file
+# include: "//looker-hub/focus_android/dashboards/*"
+# include: "views/*"
+# include: "explores/*"
+# include: "dashboards/*"
```

Comment:
Commenting those out as Looker is showing a warning since these folders do not contain any files